### ISSUE TITLE
Facebook Login Button

### ISSUE THREAD

**Issue by kong-artist** [Kong-Artist/social_reconnector#2]

**Comment by mpritham** [comment]
Why are we [checking for Facebook auth when the page is rendered?](https://github.com/Kong-Artist/social_reconnector/blob/master/web/static/js/components/Login.js#L6) Shouldn't we prompt the user to login with a FB popup?

Also how are updating the [state in the "App" component](https://github.com/Kong-Artist/social_reconnector/blob/master/web/static/js/components/App.js#L10)?

**Comment by michaelbahng999** [comment]
Yeah that's exactly what's happening, we still need the endpoint to accept
a user id when someone logs in

**Comment by mpritham** [comment]
You're right it does. I was on Safari.

Added the login endpoints. Login is '/login'. You can check what the current user's token is with '/get_token'.
